code/mob/living/carbon/human.dm

```diff
@@ -2636,6 +2642,10 @@
 			processed += organHolder.intestines
 			if (prob(25) && organHolder.intestines.loc == src)
 				ret += organHolder.intestines
+		if (organHolder.tail)
+			processed += organHolder.tail
+			if (prob(75) && organHolder.tail.loc == src)
+				ret += organHolder.tail
```

Comment:
?

Reply:
That supposedly adds the tail organ into the list of body parts that'll fly out when someone's gibbed. At least, when I set it, tails started flying out of my spaceman when they exploded.

Its set to prob(75) because that's what the other limbs were set to, and I figured that tails are _kind of_ like limbs.